narrator/parser.lua

```diff
@@ -49,8 +49,9 @@ function Parser.parse(content)
   local id = (lpeg.alpha + '_') * (lpeg.alnum + '_') ^ 0
   local label = Cg('(' * sp * C(id) * sp * ')', 'label')
   local address = id * ('.' * id) ^ -2
-  local divert = Cg(divertSign * sp * C(address), 'divert')
+  local divert = Cg( Ct(divertSign * sp * Cg(address, "path") * sp * Cg(lpeg.Cmt(Cb("path"), function(s, i, a) local r = lpeg.match (sp * divertSign, s, i) return i, r~=nil end), "tunnel") * (sp * divertSign * nl)^-1), 'divert')
```

Comment:
Do you have any ideas how to separate this long row to more small and readable sentences?

Reply:
Nope, but it feels like it needs refactoring.